server/resources/[VORP]/vorp_admin/client/database.lua

```diff
@@ -549,3 +598,42 @@ function OpenInvnetory(inventorydata)
         end)
 
 end
+
+---@param targetData: { targetID: string, moneyType: 'money' | 'gold' }
+---@return void
+function giveCurrencyToPlayer(targetData)
```

Comment:
we should remove this `giveCurrencyToPlayer` because its not being called anywhere